pkgs/development/python-modules/pycdio/default.nix

```diff
@@ -29,23 +32,29 @@ buildPythonPackage rec {
     patchShebangs .
   '';
 
+  build-system = [ setuptools ];
+
   nativeBuildInputs = [
-    nose
-    pkgs.pkg-config
-    pkgs.swig
+    pkg-config
+    swig
   ];
-  buildInputs = [
-    setuptools
-    pkgs.libcdio
-  ] ++ lib.optional stdenv.isDarwin pkgs.libiconv;
-
-  # Run tests using nosetests but first need to install the binaries
-  # to the root source directory where they can be found.
-  checkPhase = ''
-    ./setup.py install_lib -d .
-    nosetests
+
+  propagatedBuildInputs = [ libcdio ] ++ lib.optional stdenv.isDarwin libiconv;
```

Comment:
After checking the tree I think these are meant to just be `buildInputs`; sorry. (Maybe @natsukium can confirm one way or the other.)

Reply:
will wait for an definitive answer :sweat_smile:

Reply:
We’ve since confirmed that `buildInputs` is correct here. (You can just include `libiconv` unconditionally, also; it evaluates to `null` on most platforms.)